common-data/src/main/scala/org/genivi/sota/data/Interval.scala

```diff
@@ -0,0 +1,22 @@
+package org.genivi.sota.data
+
+import java.time.Instant
+
+case class Interval(start: Instant, end: Instant) {
```

Comment:
So we never write that field to the db? Otherwise we'd need a slick mapping for it.